octopusdeploy3-single-vm-windows/azuredeploy.json

```diff
@@ -0,0 +1,355 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "West US",
+      "allowedValues": [
```

Comment:
Please remove the default location. Don't believe the VMs are available in all these locations. Choosing some of these will result in an error. Please look at another template with allowed values to see the valid region list atm.